src/Select/Select.tsx

```diff
@@ -208,9 +246,62 @@ const ReactSelect = forwardRef(
     );
   }
 );
+
+const checkOptionsAreValid = (options) => {
+  if (options && process.env.NODE_ENV === "development") {
+    const uniq = (a) => Array.from(new Set(a));
+    const uniqueValues = uniq(options.map(({ value }) => (value === null ? "_null_" : value)));
+    if (uniqueValues.length < options.length) {
+      console.warn("NDS: The options prop passed to Select must have unique values for each option", options);
+    }
+  }
+};
+
+export const getOption = (options, value) => {
+  // allows an option with  a null value to be matched
+  if (options.length > 0 && value !== undefined) {
+    const optionWithMatchingValue = options.find((o) => o.value === value);
+    return optionWithMatchingValue || null;
+  }
+  return value;
+};
+
+const getReactSelectValue = (options, input) => {
+  if (Array.isArray(input)) {
+    return input.map((i) => getOption(options, i));
+  }
+  return getOption(options, input);
+};
+
+const extractValue = (options, isMulti) => {
+  if (isMulti) {
+    return options && options.length ? options.map((o) => o.value) : [];
+  }
+
+  if (options == null) {
+    return options;
+  } else {
+    return options.value;
+  }
+};
+
+const extractValuesFromCsvString = (csv: string) => {
```

Comment:
Since this is a pure function can we add some tests for it? Also, optionally can we move it to a separate file e.g: `selectUtils.ts`, `selectHelpers.ts`